src/page-multi-facility/ScenarioLibraryModal.tsx

```diff
@@ -198,6 +291,25 @@ const ScenarioLibraryModal: React.FC<Props> = ({ trigger }) => {
               );
             })
           )}
+          <ModalDialog
+            closeModal={closeDeleteModal}
+            open={showDeleteModal}
+            title="Are you sure?"
+            width="41vw"
+          >
+            <DeleteModalContents>
+              <ModalText>
+                This action can't be undone. All users currently viewing this
```

Comment:
Does this text imply that we're going to broadcast a "switch scenarios" message of some kind to other users looking at the shared scenario in the near future where sharing is enabled? I wonder if we should simplify this to avoid potentially confusing users, or having text that is subtly inconsistent with behavior.

Reply:
I don't think that was the intention.  What if instead we use: 

> This action can't be undone.  When deleting the scenario currently being viewed, users will be returned to their baseline scenario.

cc @cawarren

Reply:
That looks good to me :+1: